pkg/sypgp/sypgp.go

```diff
@@ -393,6 +393,14 @@ func GenKeyPair() (entity *openpgp.Entity, err error) {
 	if err != nil {
 		return
 	}
+	passRetype, err := AskQuestionNoEcho("Retype your passphrase : ")
```

Comment:
Would you like to offer the user the change to type the passphrase again if they don't match? For example, try three times? Otherwise the user has to enter all the information again (name, email, comment), and usually this leads to frustrated users.

Reply:
Good thinking! :+1: